package.json

```diff
@@ -1,29 +1,33 @@
 {
   "name": "omero_parade",
-  "version": "0.0.0",
+  "version": "0.1.0",
   "description": "\"React.js based centre panel for webclient\"",
-  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ome/omero-parade.git"
+  },
+  "license": "AGPL-3.0+",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node_modules/.bin/webpack-dev-server --progress --inline --hot",
-    "dev": "node_modules/.bin/webpack --config webpack.config.js --watch",
-    "build": "node_modules/.bin/webpack --progress"
+    "watch": "webpack --watch --config webpack.dev.js",
+    "build": "webpack --progress --config webpack.prod.js",
+    "build-dev": "webpack --progress --config webpack.dev.js"
   },
   "dependencies": {
-    "axios": "^0.14.0",
-    "babel-core": "^6.13.2",
-    "babel-loader": "^6.2.5",
-    "babel-preset-es2015": "^6.13.2",
-    "babel-preset-react": "^6.11.1",
+    "axios": "^0.17.1",
```

Comment:
As far as I can see, we're not using ```axios``` anywhere at the moment (just using ```jQuery``` for AJAX) so that can be removed too.
Everything is working fine for me. So, good to merge (pending minor comments)

Reply:
Once this PR is merged my preference would be to change all of the current usage of jQuery for AJAX calls to axios. In preparation for proposing the changes to the way that filters are developed I would like to minimize, or remove entirely, reliance on page dependencies not declared using `import` or `require`, `window` globals, non-ES6 style, and any tight coupling to the tree.

Reply:
Cool. Looks good to merge now then. 👍